cinch/roles/jenkins_slave/templates/swarm.service

```diff
@@ -11,10 +24,10 @@ ExecStart={{ jenkins_java_cmd }} \
           -executors "$SWARM_EXECUTORS" \
           -labels "$SWARM_SLAVE_LABEL" \
           -fsroot "$SWARM_ROOT" \
+          -retry 0 \
```

Comment:
are you sure you want to limit it so strictly? Wouldn't at least one retry make sense here?